### ISSUE TITLE
Error when starting sync

### ISSUE THREAD

**Issue by meneses-pt** [guyzyl/whatsapp-contact-sync#20]
I installed the app locally and tried to run it. When starting sync, got this error on server and it didn't move any further:

```
TypeError: Cannot read properties of undefined (reading '0')
    at /Users/meneses/git/whatsapp-contact-sync/server/src/gapi.ts:48:36
    at Array.map (<anonymous>)
    at /Users/meneses/git/whatsapp-contact-sync/server/src/gapi.ts:44:8
    at Generator.next (<anonymous>)
    at fulfilled (/Users/meneses/git/whatsapp-contact-sync/server/src/gapi.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[nodemon] app crashed - waiting for file changes before starting...
```

Can you please help?
(Will buy you a coffee if this works)

**Comment by guyzyl** [comment]
On what page did you get this error?

**Comment by andre-meneses-fivestars** [comment]
Right when I started the sync process. Where it was supposed to show the progress.

**Comment by guyzyl** [comment]
Is it possible that you have a contact in Google that doesn't have a name?
There might be a bug in the code due to that, I'll fix it this weekend and hopefully it will fix your issue.

**Comment by andre-meneses-fivestars** [comment]
It is possible that I do... Can you please let me know when you fix that?

**Comment by guyzyl** [comment]
@andre-meneses-fivestars just merged a possible fix, see if #21 works for you.

**Comment by meneses-pt** [comment]
Worked perfectly! Thank you!
